src/lib/resource/interceptor/handler.es

```diff
@@ -67,9 +67,24 @@ export class InterceptorHandler {
     this._listenOnAddEvent()
   }
 
+  /**
+   * RESPONSE_METHOD getter
+   * @static
+   * @returns {string}
+  **/
   static get RESPONSE_METHOD () { return RESPONSE_METHOD }
```

Comment:
It's the only way I found to be able to use a nice clean Class.CONSTANT from outside with es6, I'm going to do a little more research about this.